content/Setting Up Firebase on Android- Amaury Ricardo.md

```diff
@@ -0,0 +1,113 @@
+
+## Setting Up Firebase on Android.
+### What's Firebase?
+
+Firebase is a mobile platform that helps you quickly develop high-quality apps, grow your user base, and more. Firebase is made up of complementary features that you can mix-and-match to fit your needs, helping you to build apps fast, without managing infrastructure. Firebase gives you functionality like analytics, databases, messaging, and crash reporting so you can move quickly and focus on your users. [[from google]](https://developer.android.com/studio/write/firebase)
+
+Was created by Envolved between 2011-2014, with just a couple o features (Realtime db, Hosting and Authentication) and was acquired by Google in 2014. A year later, in 2015, Google acquired Divshot, an HTML5 web-hosting platform, to merge it with Firebase, and integrated a lot of google services as well, in 2016 acquired LaunchKit a toolkit for mobile app development and in 2017 Fabric as analytics tool. Every year Google release at least one new functionality for Firebase in order to improve the mobile and web development.
```

Comment:
Seems a bit too much and historic data I think its irrelevant for development purposes.